content/videos/challenges/179-wolfram-ca/index.json

```diff
@@ -3,7 +3,7 @@
   "videoNumber": "179",
   "description": "How is nature hidden in a pile of 0s and 1s? Let's find out by coding a p5.js visualization of the Wolfram Elementary Cellular Automaton!",
   "videoId": "Ggxt06qSAe4",
-  "nebulaSlug": "https://nebula.tv/videos/codingtrain-wolfram-elementary-ca",
+  "nebulaSlug": "git codingtrain-wolfram-elementary-ca",
```

Comment:
bad slug again

Reply:
@shiffman you missed this issue before merging!

Reply:
fixed in 6f9e201